### PR TITLE
feat(ios): add generic border API

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-27305

Can someone take over Android? I'll add unit tests after parity is done.

Note -  Need to merge with PR #11114 . Docs has changes for android as well.

### CODE REVIEW

**Comment by m1ga** [comment]
Android parity in this PR:https://github.com/appcelerator/titanium_mobile/pull/11114

File: apidoc/Titanium/UI/UI.yml [hunk 1]
```diff
@@ -2844,6 +2844,34 @@ properties:
     since: "6.0.0"
     platforms: [iphone, ipad]
 
+  - name: BORDER_EDGE_BOTTOM_LEFT
+    summary: A bottom left border edge used in <Titanium.UI.View.borderRadiusEdges>.
+    type: Number
+    platforms: [android, iphone, ipad]
+    since: "8.3.0"
```
Comment (m1ga) [inline apidoc/Titanium/UI/UI.yml]: version 9? :crossed_fingers: ?

**Comment by brentonhouse** [comment]
This is a big missing feature.  What would it take to get this merged?  @sgtcoolguy

**Review by vijaysingh-axway — APPROVED** [review]
CR passed.

**Comment by vijaysingh-axway** [comment]
This is continued in PR https://github.com/appcelerator/titanium_mobile/pull/11770. So closing.

**Pull request closed without merging**